Big-Spender/readme.md

```diff
@@ -68,39 +68,41 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where description ilike '%fee%';
```

Comment:
Can you use uppercase letters in the SQL commands? To be consistent with the other queries you have above.

e.g. 

SELECT * FROM spends WHERE description ILIKE '%fee%';